### ISSUE TITLE
Empty SecurityRequirement in YAML file

### ISSUE THREAD

**Issue by NukeBoy** [cebe/php-openapi#46]
When trying to write a Security Requirement object to a YAML file:
`$openapi->security = [new SecurityRequirement(['Bearer' => []])];`
The result is:
```
security:
        -
          Bearer: {  }
```
When expected:
```
security:
        -
          Bearer: []
```
When writing to a JSON file, everything is fine:
```
"security": [
    {
         "Bearer": []
    }
]
```

**Comment by cebe** [comment]
seems symfony/yaml is interpreting this wrong... not sure how to solve it, if it would print an array intead of object we could just pass a stdClass object, but I don't know a way to force the type the other way round...

**Comment by coatesap** [comment]
Potential fix for this suggested in #52
